lessons/intro/async/index.md

```diff
@@ -1,24 +1,28 @@
 
-Na část toto cvičení bude opět potřeba PyQt5.
-Můžete použít virtualenv z minula nebo PyQt5 nainstalovat znovu (viz [lekce o PyQt]).
-(Nejde-li to, nevadí – úplně nezbytné dnes PyQt nebude.)
+Ve cvičení použijeme ukázku z PyQt5.
+Máte-li eště virtualenv s nainstalovaným PyQt, použijte ho, případně ho
+podle [lekce o PyQt] nainstalujte znovu.
 
-[lekce o PyQt]: {{ lesson_url('intro/pyqt') }}
-
-Další knihovny pro dnešní den:
+K PyQt si přiinstalujte knihovnu `quamash`:
 
 ```console
-$ python -m pip install --upgrade pip
-$ python -m pip install notebook aiohttp quamash
+$ python -m pip install quamash
 ```
 
-Případně pro Python 3.3 i:
+Nejde-li to, nevadí – nezbytné dnes PyQt nebude.
+
+[lekce o PyQt]: {{ lesson_url('intro/pyqt') }}
```

Comment:
Dá se tady použít jen `../intro/pyqt/`? Případně bude to mít nějaké implikace.

Reply:
Dá, mělo by to fungovat bez problémů.